qa/L0_analyze/test.sh

```diff
@@ -1,103 +0,0 @@
-# Copyright (c) 2021 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Is there a matching MR to remove this test from CI?

Reply:
Yes, I've created this